Userland/Libraries/LibWeb/Loader/FileDirectoryLoader.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023, Bastiaan van der Plaat <bastiaan.v.d.plaat@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibWeb/Loader/Resource.h>
+
+namespace Web {
+
+class FileDirectoryLoader {
+public:
+    static FileDirectoryLoader& the();
+
+    ErrorOr<DeprecatedString> load(LoadRequest const&) const;
+};
```

Comment:
This doesn't really need to be a class, because it doesn't have any state. It would work just as well as a free function:

```c++
ErrorOr<DeprecatedString> load_file_directory_page(LoadRequest const&);
```

Reply:
That's partially my fault, sorry!

Reply:
Good point, I have fixed it.